src/richie/apps/courses/models.py

```diff
@@ -224,6 +228,61 @@ def copy_relations(self, oldinstance, language):
         self.courses.set(oldinstance.courses.drafts())
 
 
+class Licence(models.Model):
+    """
+    Licence model.
+
+    Instances of this models should only be created by administrators.
+    """
+
+    name = models.CharField(_("name"), max_length=200)
+    logo = FilerImageField(
+        related_name="licence",
+        verbose_name=_("logo"),
+    )
+    link_url = models.CharField(
```

Comment:
link and url are synonym. Proposals:
"resource_link"? 
or just "url"?

Reply:
Just "url" is perfect.